### ISSUE TITLE
Minecraft don't working on Apple silicon (M1)

### ISSUE THREAD

**Issue by MrDogPL** [minecraft-linux/mcpelauncher-manifest#604]
The game crashes at **random time**error code: 0. (I know that Minecraft works on roseta, but I guess you can fix it)Os: macos big sur 11.5.2

**Comment by kyleplo** [comment]
I'm not sure if this is possible, since this project depends on the fact that Minecraft is already compiled for Intel processors as some Android devices have Intel processors, while there are no Android devices that use Apple Silicon processors, so Minecraft has not been compiled for them.

**Comment by ChristopherHX** [comment]
> since this project depends on the fact that Minecraft is already compiled for Intel processors 

No. Linux arm32/64 works and has even less bugs in Minecraft 1.17.

Apple silicon is not emulatable, while linux / windows arm64 is.
I need access to a m1 mac (remote CI is probably enough) or there will be no progress.

**Comment by CasperOng** [comment]
The problem also occur in M2, M1 Max and M1 Ultra. I didn't test anything about M1 Pro but I guess the problem still happened.

**Comment by gopipolsani** [comment]
For me on M2 MacBook Pro 13 inch with MacOS Ventura it works flawlessly under rosseta 2. But it hogs a lot of battery since its running under rosetta, is there any native Apple Silicon support?

**Comment by iMonZ** [comment]
Hey any news here? Would like to uninstall Rosetta 2?
